Enable building of FakeLowP ops

yinghai opened the pull request:
We open sourced the FakeLowp ops as a reference implementation of fp16 ops. This PR makes it buildable. 

```
USE_CUDA=0 USE_ROCM=0 USE_FAKELOWP=ON python setup.py install
```

Test plan:
Build Onnxifi library in Glow. 
```
cp ${GLOW}/build/lib/Onnxifi/libonnxifi-glow.so ${MY_PATH}/ibonnxifi.so
LD_LIBRARY_PATH=${MY_PATH}/ibonnxifi.so python pytorch/caffe2/python/fakelowp/test_sls_nnpi_fp16.py
```

It doesn't run successfully right now because we need to open source the glow gflags and some other ops like `FbgemmPack`.

yinghai commented:
@hyuen

hyuen reviewed (approved):
Looks good, I saw some failures in some jobs, make sure you fix those

yinghai commented on cmake/Summary.cmake:
```diff
@@ -88,6 +88,7 @@ function(caffe2_print_configuration_summary)
   message(STATUS "  USE_ROCM              : ${USE_ROCM}")
   message(STATUS "  USE_EIGEN_FOR_BLAS    : ${CAFFE2_USE_EIGEN_FOR_BLAS}")
   message(STATUS "  USE_FBGEMM            : ${USE_FBGEMM}")
+  message(STATUS "    USE_FAKELOWP          : ${USE_FAKELOWP}")
```
Oh this is intentional.

yinghai committed:
Enable building of FakeLowP ops (#36170)

Summary:
We open sourced the FakeLowp ops as a reference implementation of fp16 ops. This PR makes it buildable.

```
USE_CUDA=0 USE_ROCM=0 USE_FAKELOWP=ON python setup.py install
```
Pull Request resolved: https://github.com/pytorch/pytorch/pull/36170

Test Plan:
Build Onnxifi library in Glow.
```
cp ${GLOW}/build/lib/Onnxifi/libonnxifi-glow.so ${MY_PATH}/ibonnxifi.so
LD_LIBRARY_PATH=${MY_PATH}/ibonnxifi.so python pytorch/caffe2/python/fakelowp/test_sls_nnpi_fp16.py
```

It doesn't run successfully right now because we need to open source the glow gflags and some other ops like `FbgemmPack`.

Differential Revision: D20980681

Pulled By: yinghai

fbshipit-source-id: 7b53048a825aa73c73ab072c3f2c94de08388d8a
